query/src/test/scala/hmda/query/repository/filing/FilingComponentSpec.scala

```diff
@@ -99,10 +99,10 @@ class FilingComponentSpec extends AsyncWordSpec with MustMatchers with FilingCom
     "Stream rows for a specific respondent id" in {
       val respId = "resp2"
       val p = ""
-      val lar1 = larGen.sample.get.copy(respondentId = respId)
-      val lar2 = larGen.sample.get.copy(respondentId = respId)
-      val lar3 = larGen.sample.get.copy(respondentId = respId)
-      val lar4 = larGen.sample.get.copy(respondentId = "resp3")
+      val lar1 = larGen.sample.getOrElse(LoanApplicationRegister()).copy(respondentId = respId)
```

Comment:
Code like this is repeated lots of times: `larGen.sample.getOrElse(LoanApplicationRegister())`, `InstitutionGenerators.institutionGen.sample.getOrElse(Institution.empty)`, etc.

Could it work to abstract that into a method? Like this: `LarGenerators.getOne` and `InstitutionGenerators.getOne`